prac_09/taxi.py

```diff
@@ -1,12 +1,14 @@
 # taxi.py
 from prac_09.car import Car
 
+
 class Taxi(Car):
     """Specialised version of a Car that includes fare costs."""
 
-    def __init__(self, name, fuel, price_per_km):
+    def __init__(self, name, fuel, price_per_km=1.23):
```

Comment:
Should **price_per_km = 1.23** be declared as a class variable rather than a default value?